autogpt_platform/backend/backend/executor/scheduler.py

```
@@ -1634,6 +1652,27 @@ def _job_to_info(
     return None
 
 
+def _index_entry(
+    job_id: str, args: Union[GraphExecutionJobArgs, CopilotTurnJobArgs]
```

Ruff (0.16.2)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)